devices/Bq2579x/Bq2579x.cs

```diff
@@ -261,6 +268,58 @@ public class Bq2579x : IDisposable
         [Property]
         public ThermalShutdownThreshold ThermalShutdownThreshold { get => GetThermalShutdownThreshold(); set => SetThermalShutdownThreshold(value); }
 
+        /// <summary>
+        /// Gets or sets charge voltage for JEITA high temperature range (TWARN � THOT). 
+        /// </summary>
+        [Property]
+        public ChargeVoltage ChargeVoltageHighTempRange { get => GetChargeVoltage(); set => SetChargeVoltage(value); }
+
+        /// <summary>
+        /// Gets or sets charge current for JEITA high temperature range (TWARN � THOT). 
```

Comment:
same here:
![image](https://github.com/nanoframework/nanoFramework.IoT.Device/assets/8145578/cd1fe45d-7816-4566-a1ec-1235e7766283)

Reply:
And please tell what JEITA is in the remarks or equivalent